Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -176,29 +176,29 @@ static void buildTextForShaping(ShapedContent& shapedContent, InlineIterator::Bo
 }
 
 TextBoxPainter::TextBoxPainter(const LayoutIntegration::InlineContent& inlineContent, const InlineDisplay::Box& box, const RenderStyle& style, PaintInfo& paintInfo, const LayoutPoint& paintOffset)
-    : m_textBox(InlineIterator::BoxModernPath { inlineContent, inlineContent.indexForBox(box) })
-    , m_renderer(downcast<RenderText>(m_textBox.renderer()))
-    , m_document(m_renderer.document())
-    , m_style(style)
-    , m_logicalRect(m_textBox.isHorizontal() ? m_textBox.visualRectIgnoringBlockDirection() : m_textBox.visualRectIgnoringBlockDirection().transposedRect())
-    , m_paintTextRun(m_textBox.textRun())
-    , m_paintInfo(paintInfo)
-    , m_selectableRange(m_textBox.selectableRange())
-    , m_paintOffset(paintOffset)
-    , m_paintRect(computePaintRect(paintOffset))
-    , m_isFirstLine(m_textBox.isFirstFormattedLine())
-    , m_isCombinedText([&] {
-        auto* combineTextRenderer = dynamicDowncast<RenderCombineText>(m_renderer);
-        return combineTextRenderer && combineTextRenderer->isCombined();
-    }())
-    , m_isPrinting(m_document.printing())
-    , m_haveSelection(computeHaveSelection())
+: m_textBox(InlineIterator::BoxModernPath { inlineContent, inlineContent.indexForBox(box) })
```

Comment:
I think something is fishy with the indentation here no? I think the previous indention was correct unless I'm missing something.